packages/xstate-graph/test/__snapshots__/graph.test.ts.snap

```diff
@@ -899,6 +899,323 @@ exports[`shortest paths for reducers 1`] = `
 ]
 `;
 
+exports[`shortest paths for transition functions 1`] = `
```

Comment:
hm, it feels that perhaps there are some outdated snapshots to be removed in this file